eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -381,6 +381,9 @@ the main ServiceBusClientBuilder. -->
   <suppress files="com.azure.spring[\\/].*Configuration\.java" checks="MissingJavadocMethod" />
   <suppress files="com.azure.spring[\\/].*Metadata\.java" checks="MissingJavadocMethod" />
 
+  <!-- Suppress spring-cloud-azure-native-configuration module, because of only some auto configurations. -->
+  <suppress files="[/\\]spring-cloud-azure-native-configuration[/\\]src[/\\]main[/\\]java[/\\].*[/\\]*Hints.java" checks="MissingJavadocMethod|MissingJavadocType|MissingJavadocPackage|JavadocPackage|PackageName" />
+
```

Comment:
I think we can remove this, for all of our codes are in the implementation package now.